plotly/graph_reference.py

```diff
@@ -410,12 +410,25 @@ def _patch_objects():
                          'attribute_paths': layout_attribute_paths,
                          'additional_attributes': {}}
 
-    figure_attributes = {'layout': {'role': 'object'},
-                         'data': {'role': 'object', '_isLinkedToArray': True}}
+    figure_attributes = {
+        'layout': {'role': 'object'},
+        'data': {'role': 'object', '_isLinkedToArray': True},
+        'frames': {'role': 'object', '_isLinkedToArray': True}
+    }
     OBJECTS['figure'] = {'meta_paths': [],
                          'attribute_paths': [],
                          'additional_attributes': figure_attributes}
 
+    frame_entry_attributes = {
+        'data': {'role': 'object', '_isLinkedToArray': True},
+        'group': {'role': 'info'},
+        'layout': {'role': 'object'},
+        'name': {'role': 'info'}
+    }
```

Comment:
@etpinard , trying to grok `datasrc`, that one's a little confusing for me. How do you `src` an array of *objects*?

Also, `traces` are meant to be stored in a `Grid`? I.e., it's definitely not `info`, right? (just to confirm).